lib/enum/basic_enum.rb

```diff
@@ -85,66 +97,80 @@ def element_frequency_count
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-find-all
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-select
   def elements_ending_in_er
+    collection.find_all {|word| word[-1] == "r" && word[-2] == "e"}
```

Comment:
You can do this in one go with regex, or with a range like this: `collection.select {|element| element[-2..-1] == 'er' }`